src/ntp64.rs

```diff
@@ -96,6 +96,12 @@ impl NTP64 {
         (self.0 >> 32) as u32
     }
 
+    /// Returns the total number of nanoseconds.
```

Comment:
I think we should make clear that this is a conversion, since there is a loss of precision in the conversion of the 32-bits fraction of seconds into a number of nanoseconds.
I propose this rewording: "Convert to a number of nanoseconds"